frog-utils/src/dashboards/coordinates.js

```diff
@@ -34,6 +40,52 @@ const Viewer = (props: dashboardViewerPropsT) => {
         theme={VictoryTheme.material}
         domain={{ x: [-10, 10], y: [-10, 10] }}
       >
+        {coordinateLabels.map((x, y) => {
```

Comment:
It is useless to do a map here if you are writing the code for the 4 cases right after. I would be more readable without the `map`, `switch`, `case`, ...

Reply:
This has been fixed.